qcengine/programs/psi4.py

```diff
@@ -53,14 +53,16 @@ def compute(self, input_model: 'ResultInput', config: 'JobConfig') -> 'Result':
         if psi_version > self.parse_version("1.2"):
 
             mol = psi4.core.Molecule.from_schema(input_model)
-            if mol.multiplicity() != 1:
+            if (mol.multiplicity() != 1) and ("reference" not in input_model["keywords"]):
```

Comment:
isn't `uks` a funny choice for generic, as opposed to `uhf`?

Reply:
Yea, just hit that bug and realized why @ffangliu's example was failing.

Reply:
See update.

Reply:
Oh, I see. Thanks!